components/automate-ui/src/app/components/landing/landing.component.spec.ts

```diff
@@ -23,9 +23,9 @@ describe('LandingComponent', () => {
       anyOfCheck: [['/auth/something-else', 'get', '']],
       route: '/settings/teams'
     },
-    { allOfCheck: [['/auth/users', 'get', '']], route: '/settings/users' },
+    { allOfCheck: [['/iam/v2/users', 'get', '']], route: '/settings/users' },
     {
-      anyOfCheck: [['/auth/tokens', 'get', ''], ['/iam/v2/tokens', 'get', '']],
+      anyOfCheck: [['/iam/v2/tokens', 'get', ''], ['/iam/v2/tokens', 'get', '']],
```

Comment:
I think this is covered in @blakestier's PR? Also it's repeating the same path

Reply:
This is in my PR (and is repeating the same path)